packages/thumbprint-react/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Changed
+
+-   [Patch] Convert `Alert` component to React (TypeScript).
```

Comment:
```suggestion
-   [Minor] Convert `Alert` component to React (TypeScript).
```

This would be "Minor" in SemVer since it's the addition of new functionality.